dhcpv4/ztpv4/parse_circuitid_test.go

```diff
@@ -85,6 +86,7 @@ func TestParseCircuitID(t *testing.T) {
 		{name: "Cisco pattern", circuit: []byte("Gi1/10:2020"), want: &CircuitID{Slot: "1", Port: "10", Vlan: "2020"}},
 		{name: "Cisco Nexus pattern", circuit: []byte("Ethernet1/3"), want: &CircuitID{Slot: "1", Port: "3"}},
 		{name: "Juniper Bundle Pattern", circuit: []byte("ae52.0"), want: &CircuitID{Port: "52", SubPort: "0"}},
+		{name: "Arista Vlan pattern 1 with SHIFT IN", circuit: bytes.Join([][]byte{[]byte{0x00, 0x0f}, []byte("Ethernet14:Vlan2001")}, []byte{}), want: &CircuitID{Port: "14", Vlan: "Vlan2001"}},
```

Comment:
What is the `[]byte{}` for in the last parameter of bytes.Join?

Reply:
it's the separator used to join the two byte sequences - empty separator, basically it concatenates.

However I would recommend to change the whole call to `bytes.Join` to a simpler `[]byte("\x00'x0fEthernet14:Vlan2001")`